addons/base_vat/models/res_partner.py

```diff
@@ -189,6 +192,12 @@ def check_vat_ch(self, vat):
             return check == int(num[8])
         return False
 
+    def check_vat_cl(self, vat):
```

Comment:
Please move at the very end of the file.

Reply:
ok

Reply:
@nim-odoo Done
Move at the end of the file